docs/reference/go/api.md

```diff
@@ -29,27 +29,17 @@ For more technical information about building, debugging or using the API, see:
 If you plan to use Kubo as a package in your own Go application, you can take any of three main approaches:
 
 - Use [kubo](https://github.com/ipfs/kubo) to run Kubo IPFS directly in your own process.
-- Use [kubo RPC client](https://github.com/ipfs/go-ipfs-http-client) to communicate with a Kubo IPFS daemon in a separate process via its HTTP RPC API (this is what Kubo does if a daemon is already running).
+- Use [kubo RPC client](https://pkg.go.dev/github.com/ipfs/kubo/client/rpc) to communicate with a Kubo IPFS daemon in a separate process via its HTTP RPC API (this is what Kubo does if a daemon is already running).
 - Use other Go packages to communicate with the HTTP RPC API directly. See the [RPC API reference](../../reference/kubo/rpc.md).
 
-## Go CoreAPI
+## Go Embedded Client
 
-[CoreAPI interfaces](https://godoc.org/github.com/ipfs/interface-go-ipfs-core) for go-ipfs. Package iface defines IPFS Core API, which is a set of interfaces used to interact with IPFS nodes.
+[Package `coreapi`](https://godoc.org/github.com/ipfs/kubo/core/coreapi) provides direct access to the core commands in IPFS. If you are embedding IPFS directly in your Go program, this package is the public interface you should use to read and write files or otherwise control IPFS. **This package is experimental and subject to change.**
 
-## Go embedded client
-
-[Package coreapi](https://godoc.org/github.com/ipfs/kubo/core/coreapi) provides direct access to the core commands in IPFS. If you are embedding IPFS directly in your Go program, this package is the public interface you should use to read and write files or otherwise control IPFS. **This package is experimental and subject to change.**
-
-If you are running IPFS as a separate process, you should use `go-ipfs-api` to work with it via RPC. As we finalize the interfaces in this embedded client, `go-ipfs-api` will transparently adopt them so you can use the same code with either package.
-
-## Go HTTP clients
-
-[CoreAPI implementation using HTTP API](https://godoc.org/github.com/ipfs/go-ipfs-http-client). **This package is experimental and subject to change.** For an old but stable Go HTTP client, use [CoreAPI interfaces](https://godoc.org/github.com/ipfs/interface-go-ipfs-core).
+If you are running Kubo as a separate process, you should use the [Kubo RPC Client](https://pkg.go.dev/github.com/ipfs/kubo/client/rpc) to work with it via RPC.
 
 ## Packages
 
-> This table is generated using the module [`package-table`](https://github.com/ipfs-shipyard/package-table) with `package-table --data=package-list.json`.
-
```

Comment:
ℹ️ I removed this because (1) this list is not generated by `package-table` anymore since https://github.com/ipfs/ipfs-docs/commit/94597903d35839aca3d7c86e195d83ed0f263e1b, and (2) there's no `package-list.json` in this repository to generate it.